uiowa/uiowa
# Remove permissions for adding new instagram media.

Related to #7595 

# How to test
- sync a site
- masquerade as a webmaster or lower
- try to add a new Instagram media item, via the WYSIWYG or via /media/add/. See that Instagram is no longer an option.
- Review https://sitenow.uiowa.edu/known-issues#instagram-embeds. (I just added one sentence to the end of the issue description.)
